server/dist/models/list_model.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+var __importDefault = (this && this.__importDefault) || function (mod) {
+    return (mod && mod.__esModule) ? mod : { "default": mod };
+};
+Object.defineProperty(exports, "__esModule", { value: true });
+const mongoose_1 = __importDefault(require("mongoose"));
+const list_item_model_1 = __importDefault(require("./list_item_model"));
+const { Model, Schema } = mongoose_1.default;
+const listSchema = new Schema({
+    date: { type: Date, required: true },
+    items: { type: [list_item_model_1.default], required: true },
+    story: { type: String, required: false }
+});
```

Comment:
We also need an optional image field that stores the URL to the image corresponding to their story